src/asmcnc/apps/drywall_cutter_app/screen_drywall_cutter.py

```diff
@@ -197,13 +198,16 @@ def __init__(self, **kwargs):
         self.jd = kwargs['job']
         self.pm = kwargs['popup_manager']
 
-        self.engine = GCodeEngine(self.m, self.dwt_config)
+        self.cs = CoordinateSystem(self.m)
```

Comment:
Could this belong to router_machine? Would make more sense when using it if it was self.machine.coordinates/self.machine/coordinates_system

Reply:
I agree it should be higher level but in the interest of time I think this should be a later refactor